.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
ufff... my brain is not working fine today hahaha. I forgot that I've already pushed a commit that changed the value for package.json "main" property.